test/plugin.test.ts

```diff
@@ -51,6 +52,30 @@ test('default options', async () => {
 	expect(result['index.html']).toMatchSnapshot();
 });
 
+test('mini-css-extract-plugin', async () => {
+	const result = await compile({
+		mode: 'production',
+		entry: { main: './test/fixtures/index.js' },
+		module: {
+			rules: [
+				{
+					test: /\.css$/,
+					use: [MiniCssExtractPlugin.loader, 'css-loader'],
+				},
+			],
+		},
+		plugins: [
+			new MiniHtmlWebpackPlugin({}),
+			// @ts-ignore: MCEP TS declarations are broken
```

Comment:
Save couple of brain cycles:

```suggestion
			// @ts-ignore: MiniCssExtractPlugin types are broken
```